packages/@lwc/perf-benchmarks/package.json

```diff
@@ -28,7 +28,7 @@
         "@best/runner-remote": "^8.1.3",
         "@lwc/rollup-plugin-node-resolve-legacy": "npm:rollup-plugin-node-resolve@5.2.0",
         "@lwc/rollup-plugin-commonjs-legacy": "npm:rollup-plugin-commonjs@10.1.0",
-        "glob-hash": "^1.0.5",
+        "folder-hash": "4.0.4",
```

Comment:
This is a category-A and meets the requirements for auto-approval. However, I couldn't find in the FAQ where to actually submit this for approval. Does anyone know if that's needed?

Reply:
Are you asking if we need to formally request approval to add a dependency? No, we've never done that before.